tests/integration/testsuites/gateway/features/gateway.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Checking default kyma gateway
+  Scenario: Kyma gateway is deployed
```

Comment:
This scenario is specific for open source Kyma, right? I mean the resources also apply for Gardener, but in my opinion tests are also documentation of the behaviour. With this test it's not clearly visible, that only k3d must have only this resources.

Reply:
This scenario if for both OS and Gardener.

Reply:
I don't agree, it's only by accident also working for Gardener, because some of the resource are the same and Open Source doesn't have any additional resources. From a test perspective we are not clearly separating Gardener and Open source test cases. I don't like it.

Reply:
Added separate feature for k3d